src/__tests__/fs.spec.js

```diff
@@ -108,6 +108,15 @@ describe('copyFiles()', () => {
 		expect(vol.toJSON()).toEqual(json);
 	});
 
+	it('should throw if contents is the same and errorOnExist=true', () => {
```

Comment:
contents is the same → file already exists?

Reply:
Yes